examples/getting_started/custom_parameters.py

```diff
@@ -131,21 +155,37 @@ def prepare_ocp(
     # Give the parameter some min and max bounds
     parameters = ParameterList()
     bound_gravity = Bounds(min_g, max_g, interpolation=InterpolationType.CONSTANT)
+    bound_mass = Bounds(min_m, max_m, interpolation=InterpolationType.CONSTANT)
     # and an initial condition
     initial_gravity = InitialGuess((min_g + max_g) / 2)
+    initial_mass = InitialGuess((min_m + max_m) / 2)
     parameter_objective_functions = Objective(
-        my_target_function, weight=10, quadratic=True, custom_type=ObjectiveFcn.Parameter, target=target_g
+        my_target_function, weight=1000, quadratic=True, custom_type=ObjectiveFcn.Parameter, target=target_g
     )
-    parameters.add(
-        "gravity_z",  # The name of the parameter
-        my_parameter_function,  # The function that modifies the biorbd model
-        initial_gravity,  # The initial guess
-        bound_gravity,  # The bounds
-        size=1,  # The number of elements this particular parameter vector has
-        penalty_list=parameter_objective_functions,  # ObjectiveFcn of constraint for this particular parameter
-        extra_value=1,  # You can define as many extra arguments as you want
+    mass_objective_functions = Objective(
+        my_target_function, weight=100, quadratic=True, custom_type=ObjectiveFcn.Parameter, target=target_m
     )
-
+    if optim_gravity:
+        parameters.add(
+            "gravity_xyz",  # The name of the parameter
+            my_parameter_function,  # The function that modifies the biorbd model
+            initial_gravity,  # The initial guess
+            bound_gravity,  # The bounds
+            size=3,  # The number of elements this particular parameter vector has
+            penalty_list=parameter_objective_functions,  # ObjectiveFcn of constraint for this particular parameter
+            scaling=np.array([1.0, 1.0, 10.0]),
+            extra_value=1,  # You can define as many extra arguments as you want
+        )
+    if optim_mass:
+        parameters.add(
+            "mass",  # The name of the parameter
+            set_mass,  # The function that modifies the biorbd model
+            initial_mass,  # The initial guess
+            bound_mass,  # The bounds
+            size=1,  # The number of elements this particular parameter vector has
+            penalty_list=mass_objective_functions,  # ObjectiveFcn of constraint for this particular parameter
+            scaling=np.array([10.0]),
+        )
```

Comment:
I think it would be slightly easier to read if the gravity parameters were regroup and then mass
Good Idea to add a parameter that has 3 components